tests/system_tests_one_router.py

```diff
@@ -976,7 +976,6 @@ def test_50_extension_capabilities(self):
         """
         client_caps = [symbol("single"),
                        ["A", "B", "C"],
-                       [9, 10, 11],
                        [],
                        None]
```

Comment:
`None` is OK?

Reply:
`None`(null) seems to be allowed but I would defer to @astitcher

Reply:
The test works with it, which is what matters here.